.github/workflows/cron-pins.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         env: ['staging', 'production']
-    timeout-minutes: 30
```

Comment:
it will run each 30 minutes, so we will start having multiple jobs at same time. See line 5